### PR TITLE
gtk-doc-stub: Support Conan V2

### PR DESCRIPTION
**Pull request by jwillikers**
Specify library name and version:  **gtk-doc-stub/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/gtk-doc-stub/all/conanfile.py [hunk 1]
```diff
@@ -12,54 +16,60 @@ class GtkDocStubConan(ConanFile):
...
 
     def package(self):
-        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
-        autotools = self._configure_autotools()
+        copy(self, "COPYING", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        autotools = Autotools(self)
         autotools.install()
 
     def package_info(self):
+        self.cpp_info.bindirs = ["bin"]
```
[the remark this answers is not in the document]
Comment (jwillikers) [reply recipes/gtk-doc-stub/all/conanfile.py:63]: I have trouble keeping track sometimes :)

[the remark this answers is not in the document]
Comment (RubenRBS) [reply recipes/gtk-doc-stub/all/conanfile.py]: Hah, totally understandable!

**Comment by RubenRBS** [comment]
Taking a look into the v2 issue now :)

**Comment by RubenRBS** [comment]
See https://github.com/jwillikers/conan-center-index/pull/11 and merge if it looks ok to you @jwillikers :D

**Comment by RubenRBS** [comment]
Failing for v1 now 🤦

**Comment by RubenRBS** [comment]
@jwillikers sorry it took me a bit to get back to this, I've now pushed the necessary changes to your branch directly, thanks for your patience :)

**Review by czoido — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Comment by jwillikers** [comment]
@RubenRBS Awesome! Thank you!

**Pull request merged**